src/bsp/include/bsp/BSPContainer.h

```diff
@@ -25,7 +26,7 @@ namespace BSPContainer {
     IHostUart& getHostUart();
 
     /**
-     * @brief Returns an instance of the CRC driver.
+     * @brief Returns an instance of the UserCRC driver.
```

Comment:
UserCRC? it's the crc that is not used internally and that the user can use?

Reply:
It's the CRC defined by the user (us) if you have a better name I can change it

Reply:
HCRC or HardwareCRC maybe?